Composer/packages/client/src/components/CreationFlow/v2/DefineConversation.tsx

```diff
@@ -64,6 +64,11 @@ const stackinput = {
   ],
 };
 
+const RuntimeLanguageOptions: IDropdownOption[] = [
```

Comment:
camelCase the name and make it a function, otherwise it wont pick up locale changes for node option

Reply:
you can also move it inside the component and use useMemo

Reply:
handled